src/BlueSCSI_disk.cpp

```diff
@@ -544,7 +550,7 @@ class ImageBackingStore
 
     void flush()
     {
-        if (!m_israw && !m_isrom)
+        if (!m_israw && !m_isrom && !m_isreadonly_attr)
```

Comment:
Could also be expressed this way
`if (! (m_israw || m_isrom || m_isreadonly_attr))`